### PR TITLE
Concf 403

### PR DESCRIPTION
**Pull request by hakubo**
JSMessages called in setup is too early unless I would change logic in DefaultSettings
JSMessages called in init of WikiaMobileService is too late to work

including it in CommonExtensions is way to go, we have everything at this stage to work properly

@rogatty @bognix @vforge @SebastianMarzjan - will that work?

### CODE REVIEW

File: extensions/wikia/WikiaMobile/WikiaMobile.setup.php [hunk 1]
```diff
@@ -76,6 +76,28 @@
  */
 $wgExtensionMessagesFiles['WikiaMobile'] = "{$dir}/WikiaMobile.i18n.php";
 
+// initialize i18ns
+JSMessages::registerPackage( 'WkMbl', array(
```
Comment (Warkot) [inline extensions/wikia/WikiaMobile/WikiaMobile.setup.php]: I would swap `array()` for `[]` to keep it consistent.

**Comment by bognix** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**